### ISSUE TITLE
vale doesn't detect .vale.ini

### ISSUE THREAD

**Issue by c0rn1ie** [jdkato/txtlint#511]
- Linux
- AUR
- 2.20.1

`vale` doesn't detect my `.vale.ini` file as demonstrated in https://vale.sh/docs/vale-cli/structure/#sample-repository without specifying `--config=.vale.ini`. Is the documentation wrong?

**Comment by jdkato** [comment]
You do not need to specify a path with `--config`. You can verify what config Vale is using by running `vale ls-config`.

**Comment by c0rn1ie** [comment]
`vale ls-config` says:

```
E100 [--config] Runtime error

path '/home/user/.config/vale/vale.ini' does not exist

Execution stopped with code 1
```

I have a `.vale.ini` file in my current directory per the above link.

**Comment by jdkato** [comment]
If Vale was failing to find a config file, it would report:

```
E100 [.vale.ini] Runtime error

open : no such file or directory

Execution stopped with code 1.
```

This seems to be a local configuration issue.
